### PR TITLE
Details popup

### PR DESCRIPTION
**Pull request by dagmawiii03**
**General requirements**

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**Project requirements**

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears
- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- After that, you could implement the popup window.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -271,6 +271,24 @@ span {
...
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
```
Comment (AlphaNtihinduka) [inline css/styles.css]: - [ ] We should try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=u1ymjURYn86jZtma-0). The popup should follow the design for both desktop and mobile versions. Therefore I suggest that the popup width is less than 100% and has a background blur effect. In addition, I suggest that you ensure that there is no horizontal scroll in all the sizes of the application ☺️ .

Your design 😎 
image

Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team 🙋‍♂️
Good job working on popup window. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required changes ♻️ 
- [ ] Kindly ensure that `MIT` link shows the lincense permission. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose. Please ensure that the link is not broken.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: index.js [hunk 2]
```diff
@@ -14,9 +15,150 @@ btnCloseNav.addEventListener('click', () => {
...
+      tech1: 'HTML',
+      tech2: 'CSS',
+      tech3: 'Javasctipt',
+    },
+    linkVersion: 'https://www.google.com',
+    linkSource: 'https://github.com/hendridg/',
+  },
+  {
+    id: 5,
+    url: './assets/Snapshot-Portfolio.png.jpg',
```
Comment (alexander16108) [inline index.js:98]: optional: please fix this typo error here ⏫ 😄

**Review by alexander16108 — APPROVED** [review]
Hi @dagmawiii03 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Gitflow was correctly used. ✔️
- Good commit messages ✔️
- Professional README ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (dagmawiii03) [reply index.js:98]: Thanks for the feedack

**Pull request merged**